boggle.js

```diff
@@ -6,26 +25,136 @@ function newBoggleBoard() {
     ['_', '_', '_', '_'],
   ];
 }
-
-/**
- * Prints out a boggle board.
- */
 function printBoggleBoard(boggleBoard) {
+  let isQ = false;
+  for (let i = 0; i < boggleBoard.length; i++) {
+    for (let j = 0; j < boggleBoard[i].length; j++) {
+      if (boggleBoard[i][j] === 'Q') {
```

Comment:
Printing the board shouldn't modify the board itself. If you checked for a word and then printed, you couldn't check for a word again because now some of the cells on the board contain spaces!